src/mms.directives.js

```diff
@@ -71,12 +71,14 @@
     angular.module('exampleApp', ['mms', 'mms.directives']);
     </pre>
  */
-angular.module('mms.directives', ['mms', 'mms.directives.tpls', 'ui.bootstrap', 'angular-growl', 'angularjs-dropdown-multiselect'])
-.config(['$sceProvider', 'growlProvider', function($sceProvider, growlProvider) {
+angular.module('mms.directives', ['mms', 'mms.directives.tpls', 'ui.bootstrap', 'angular-growl', 'angularjs-dropdown-multiselect', 'ui.tree-filter'])
+.config(['$sceProvider', 'growlProvider', 'uiTreeFilterSettingsProvider', function($sceProvider, growlProvider, uiTreeFilterSettingsProvider) {
     $sceProvider.enabled(false);
     growlProvider.onlyUniqueMessages(false);
     growlProvider.globalTimeToLive({success: 5000, error: -1, warning: 5000, info: 5000});
     growlProvider.globalPosition('bottom-right');
+    uiTreeFilterSettingsProvider.addresses = ['label'];
```

Comment:
can use branch.data.name instead of label, since that's what we actually show in the tree

Reply:
for some reason the presentation elements stop working with the filter when i use 'branch.data.name' going to leave label for now